src/app/pages/news/filters/filters.modal.ts

```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+import {ModalController, NavParams} from '@ionic/angular';
+import {FormGroup, FormBuilder, FormArray} from '@angular/forms';
+import {EnumValues} from 'enum-values';
+import { NTypes } from '@core/types';
+
+@Component({
+    templateUrl: 'filters.modal.html',
+    styleUrls: ['filters.modal.scss'],
+})
+export class FiltersModalComponent {
+    public filters = EnumValues.getValues(NTypes.ECategory);
+
+    constructor(private navParams: NavParams, private fb: FormBuilder, private modal: ModalController) {
+        console.log(navParams.data);
```

Comment:
debug code

Reply:
today you debug code tomorrow have sex with guy